build/KoreBuild.ps1

```diff
@@ -68,6 +68,22 @@ function InstallSharedRuntime([string] $version, [string] $channel)
     }
 }
 
+function BuildTaskProject {
+    $taskProj = "$repoFolder/build/tasks/RepoTasks.csproj"
+    $publishFolder = "$repoFolder/build/tasks/bin/publish/"
+
+    if (!(Test-Path $taskProj)) {
+        return
+    }
+
+    if (Test-Path $publishFolder) {
+        Remove-Item $publishFolder -Recurse -Force
```

Comment:
The reason for deleting is that `/t:publish` does not clear out the publish destination on its own, even if there is an update. This could potentially lead to stale files in the publish dir that might lead to assembly resolution issues.

I'm not actually sure if publish is incremental, but fortunately this doesn't break CSC's incremental build step.